WooCommerce/Classes/Authentication/Application Password/ApplicationPasswordTutorial.swift

```diff
@@ -78,9 +78,13 @@ struct ApplicationPasswordTutorial: View {
                     .frame(maxWidth: .infinity, alignment: .leading)
                     .padding([.horizontal, .top])
 
-                Image(uiImage: .appPasswordTutorialImage)
-                    .resizable()
-                    .aspectRatio(contentMode: .fit)
+                HStack(spacing: .zero) {
+                    Image(uiImage: .appPasswordTutorialImage)
+                        .resizable()
+                        .aspectRatio(contentMode: .fit)
+                        .frame(maxWidth: Layout.imageMaxWidth)
+                    Spacer()
```

Comment:
I'm unsure about pushing the image to the left - it does look better on larger screen size but on iPhones the image might look better when center-aligned.

Reply:
This is a comparison of how they appear pushing the image to the left. They looks good to me. Wdyt? Should I come back to the center alignment, or maybe have a custom rule just for iPad?

| iPhone 15 Pro             |  iPhone 15 Pro Max | iPad Pro 11-inch |
:-------------------------:|:-------------------------:|:-------------------------:
![Screenshot 2024-05-17 at 11 48 52](https://github.com/woocommerce/woocommerce-ios/assets/495617/0c4a0b61-53b4-4f2c-97b7-604690df73e1) | ![Screenshot 2024-05-17 at 11 59 20](https://github.com/woocommerce/woocommerce-ios/assets/495617/26c64cea-9481-43e2-8aeb-ca4eaace98ae) | ![Screenshot 2024-05-17 at 11 49 13](https://github.com/woocommerce/woocommerce-ios/assets/495617/dbdccbc5-568f-4f13-a87c-a72965559219)

Reply:
I think keeping the image in the center looks in small screen sizes, while on large screens it's less distracting to be left-aligned. If it's not too complicated, having a separate alignment on large screens would be nice, otherwise I think center-aligned image looks better.

Reply:
Applied the change here: 06edacc45656906c86911a455284e47a6c218a90

Ready for another check 🙌